app/views/shared/fields/_copyright.html.erb

```diff
@@ -24,7 +26,16 @@
 			<p><strong><%= copyright['status'].gsub(/--/, ' – ') %></strong></p>
 		<% end %>
 		<p><%= copyright['purposeNote'] %></p>
-		<p><%= copyright['note'] %></p>
+    
+    <% if unit_code == "rdcp" %>
+			<% if rights_holder && rights_holder == "UC Regents"%>
+			  <p>This work is protected by U.S. Copyright Law (Title 17, U.S.C.), and made available under a Creative Commons (CC) license. Use of this work beyond that allowed by "fair use" or the CC license requires written permission of UC San Diego. Responsibility for obtaining permissions and any use and distribution of this work rests exclusively with the user and not the UC San Diego Library. Inquiries can be made to the UC San Diego Library program having custody of the work.</p>
```

Comment:
As far as I can tell, the difference in text here, is `UC Regents` gets a `...written permission of UC San Diego` string as opposed to `...written permission of the copyright holder(s)` when the rights holder is anything else (but still an RDCP record).

Is it worth considering pulling this out into a variable, or even a small function?

Reply:
Thanks for the suggestion, I've updated it to put the differential into variable.